app/scripts/modules/core/src/pipeline/config/stages/executionWindows/ExecutionWindowExecutionDetails.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+
+import { ExecutionDetailsSection, IExecutionDetailsSectionProps } from 'core/pipeline/config/stages/core';
+import { StageExecutionLogs, StageFailureMessage } from 'core/delivery/details';
+import { ExecutionWindowActions } from 'core/pipeline/config/stages/executionWindows/ExecutionWindowActions';
```

Comment:
`from ./ExecutionWindowActions` works, we don't get paid by the character anymore.